contrib/terraform/gcp/kube_configurations/GenerateInventoryFile.py

```diff
@@ -0,0 +1,77 @@
+'''
+@author: Vinayaka V Ladwa
+@email : vinayakladwa@gmail.com
+@version : v1
+'''
+__author__ =""
+
+## This script is used to generate the kubespray inventory file#
+import logging
+import sys
+import itertools
+import json
+import traceback
+
+logging.basicConfig(
+    format='%(asctime)s:%(thread)d:%(levelname)s:%(message)s',
+    level=logging.INFO, filename="GenerateInventoryFile.log"
+)
+
+def main():
+    minion_node_details = []
+    etcd_node_details = []
+    master_node_details = []
+    try:
+        logging.info("Opening the terraform generated Inventory file for reading")
+        with  open(sys.argv[1]+"/inventory_hosts.ini","r") as file_des:
+            host_details = {}
+            for i in file_des:
+                input_line = i.rstrip('\n')
+                env,component,instance_url = input_line.split(',')
+                instance_details = instance_url.split('/')
+                project = instance_details[6]
+                instance_name = instance_details[-1]
+                instance_fqdn = "%s.c.%s.internal" %(instance_name,project)
```

Comment:
This would make a `foo.c.bar.internal` as `inventory_name`? Why not set that directly as the `template_name` in Terraform? It would be nice to have the same names between Ansible and GCP.

Reply:
Incase of cloud provider gce, we need to have nodename as the host fqdn.